### DISCUSSION TITLE
omni.isaac.orbit.controllers.differential_inverse_kinematics

### DISCUSSION THREAD

**Discussion by ZisongXu** [isaac-sim/IsaacLab#166] [Q&A] [answered]
Hi Dear:

When I tried to understand the code of "play_ik_control.py", I found a package called "omni.isaac.orbit.controllers.differential_inverse_kinematics". But I did not find this package in the orbit document. Where can I find it and learn how to use it?

Thanks a lot!

**Comment by Mayankm96** [comment, marked as answer]
Hi, sorry for the late response.

The example has been moved to tutorials: https://github.com/NVIDIA-Omniverse/orbit/blob/main/source/standalone/tutorials/05_controllers/run_diff_ik.py
